pyee/_base.py

```diff
@@ -131,18 +139,24 @@ def g(*args, **kwargs):
         else:
             return _wrapper(f)
 
+    def _remove_listener(self, event, f):
+        """Naked unprotected removal."""
```

Comment:
I'd be curious to see if this shows up in the sphinx docs - the makefile has tasks for this if you wanna give it a shot. If it shows up I'll want to remove this docstring, but otherwise this seems ok to me - though I will say, the word naked makes me giggle a little.

Reply:
I've verified that the this docstring doesn't get pulled into the sphinx docs. It seems Sphinx ignores all "private" methods / functions unless `:private-members:` is specified for the class

Reply:
Nice!